### PR TITLE
Add keras-tuner

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/keras-tuner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - future
+    - numpy
+    - python >=3.6
+    - requests
+    - scikit-learn
+    - scipy
+    - tabulate
+    - terminaltables
+    - tqdm
+    - tensorflow ==2.0
```
Comment (scopatz) [inline recipes/keras-tuner/meta.yaml]: Can this be less tightly pinned?

Comment (oblute) [reply recipes/keras-tuner/meta.yaml]: it has to be at least 2.0, testing if it can be >=. the repo says its for tf version 2.0.

Comment (oblute) [reply recipes/keras-tuner/meta.yaml]: @scopatz passes with the less strict pin.

Comment (scopatz) [reply recipes/keras-tuner/meta.yaml]: Great!

File: recipes/platformio/meta.yaml [hunk 2]
```diff
@@ -1,53 +0,0 @@
-{% set name = "platformio" %}
```
Comment (scopatz) [inline recipes/platformio/meta.yaml]: Please don't remove other files in other recipes

File: recipes/keras-tuner/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    # - tensorflow ==2.0
```
Comment (scopatz) [inline recipes/keras-tuner/meta.yaml]: Please remove comments

File: recipes/platformio/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/platformio/meta.yaml]: This file is still removed :frowning_face:

**Comment by oblute** [comment]
@rluria14 @Tata17 agree to maintain?

**Comment by Tata17** [comment]
Agree to maintain

File: recipes/platformio/meta.yaml (hunk 2 above)
Comment (oblute) [reply recipes/platformio/meta.yaml]: @scopatz i added it back.

**Pull request merged**